inspirehep/modules/records/mappings/records/hep.json

```diff
@@ -281,6 +281,9 @@
                         },
                         "volume": {
                             "type": "string"
+                        },
+                        "book_series_suggest": {
```

Comment:
Since, this field is inside `book_series`, adding the prefix `book_series` is duplicating information. 
I would suggest having that solely as `suggest`.